taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpRequest
```

Comment:
The import of `HttpRequest` is unnecessary since it's not used explicitly in the function. You can remove this import to clean up the code.